StaticWebsite/views.py

```diff
@@ -34,3 +36,33 @@ def contact(request):
 
     contact_form = ContactForm()
     return render(request, "MembershipApp/contact.html", context={"contact_form": contact_form})
+
+
+def display_events(request):
+    events = bubblyevents.objects.all()
+    context = {
+        "events": events
+    }
+    return render(request, "MembershipApp/display_events.html", context)
+
+
+@leadership_required(login_url="/member/login/")
+def leader_dashboard(request):
+    return render(request, "MembershipApp/leader_profile.html")
+
+
+def create_event(request):
+    if request.method == "POST":
+        event_form = CreateEvent(request.POST)
+        if event_form.is_valid():
+            event_form.save()
+            return redirect("/display_events")
+        else:
+            return render(
```

Comment:
we should display an error to the user 
you can use the django messages module to do that